diff --git a/node_modules/handlebars/lib/index.js b/node_modules/handlebars/lib/index.js
index 7b8b408..46e01a8 100644
--- a/node_modules/handlebars/lib/index.js
+++ b/node_modules/handlebars/lib/index.js
@@ -1,5 +1,6 @@
 // lib/index.js
 /* eslint-disable */
+/*
 var base = require('./handlebars/base');
 
 // Each of these augment the Handlebars object. No need to setup here.
@@ -16,3 +17,21 @@ require('./handlebars/helpers');
 require('./handlebars/partials');
 require('./handlebars/compiler/code-gen');
 module.exports = base;
+*/
+
+var base = require('./handlebars/base');
+
+// Each of these augment the Handlebars object. No need to setup here.
+var comp = require('./handlebars/compiler/compiler');
+var runtime = require('./handlebars/runtime');
+
+var cjs = require('./handlebars/compiler/javascript-compiler');
+
+// Environment provides an empty object that can be used to pass
+// references around and avoid circular dependencies.
+var env = base.createFrame(runtime);
+env.helpers = base.helpers;
+env.partials = base.partials;
+env.decorators = base.decorators;
+
+module.exports = base;

