diff --git a/node_modules/wav/index.js b/node_modules/wav/index.js
index 92c892b..307ed58 100644
--- a/node_modules/wav/index.js
+++ b/node_modules/wav/index.js
@@ -4,8 +4,3 @@
 exports.Writer = require('./lib/writer');
 exports.Reader = require('./lib/reader');
 exports.Header = require('./lib/header');
-
-var exts = [ 'wav', 'wave' ];
-exts.forEach(function (ext) {
-  require.extensions['.' + ext] = require;
-});
